app/models/concerns/rad_company.rb

```diff
@@ -8,12 +8,15 @@ module RadCompany
 
     scope :by_id, -> { order(:id) }
 
-    schema_validations except: :valid_user_domains
+    schema_validations except: %i[twilio_phone_numbers valid_user_domains]
+
+    before_validation :sanitize_twilio_numbers
 
     validates_with EmailAddressValidator, fields: %i[email]
     validates_with PhoneNumberValidator
     validate :validate_only_one, on: :create
     validate :validate_domains
+    validates :twilio_phone_numbers, presence: true, if: -> { RadicalTwilio.twilio_enabled? && Company.has_attribute?(:twilio_phone_numbers) }
```

Comment:
In hindsight, I think we should move twilio_phone_numbers from the database and put into an env var, and if only a single phone number is needed, the format would be the same, there would just be one